opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -348,17 +307,17 @@ components:
       schema:
         type: string
 
-    redirectCode:
+    X-XSRF-TOKEN:
       name: redirectCode
```

Comment:
Can we use same name and declaration please, it is confusing to have redirectCode and `X-XSRF-TOKEN` both

Reply:
changed to xXsrfToken